build/internal/flipt.go

```diff
@@ -86,9 +85,15 @@ func Base(ctx context.Context, dag *dagger.Client, source, uiDist *dagger.Direct
 		source.File("./ui/index.dev.html"),
 	})
 
-	// TODO(georgemac): wire in commit and version ldflags
+	gitCommit, err := golang.WithExec([]string{"git", "rev-parse", "HEAD"}).Stdout(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("getting git commit: %w", err)
+	}
+
+	// TODO(georgemac): wire in version ldflag
+	// TODO(mark): does version make sense now?
```

Comment:
how do you mean does it make sense?

Reply:
Are you asking your past self, or?

Reply:
lol idk why I thought you added this comment